docs/migration/5_0.md

```diff
@@ -381,3 +381,7 @@ The following keys have been added to `configurator-common.ts`:
 #### Template changes
 
 - Top element `<div>` has been changed to `<sector>` to make accesibility navigation consistent. 
+
+### BadRequestHandler
+
```

Comment:
You also need to add it to REMOVED_PUBLIC_API_DATA. Please check "schematics/src/migrations/5.0".

Reply:
done